OutOfSchool/OutOfSchool.WebApi/Services/ChildService.cs

```diff
@@ -246,6 +251,11 @@ public async Task<ChildDto> UpdateChildCheckingItsUserIdProperty(ChildDto childD
             throw new UnauthorizedAccessException($"User: {userId} is trying to update not his own child. Child Id = {childDto.Id}");
         }
 
+        if (child.IsParent || childDto.IsParent)
```

Comment:
Why should we check both conditions? Are the child and childDto is not the same object?

Reply:
Because we can't set child with IsParent manually and can't change excisting entity with same property.

Reply:
Test were fixed )) Thx